pennylane/drawer/draw.py

```diff
@@ -239,6 +245,10 @@ def draw_mpl(
         show_all_wires (bool): If True, all wires, including empty wires, are printed.
         decimals (int): How many decimal points to include when formatting operation parameters.
             Default ``None`` will omit parameters from operation labels.
+        style (str): visual style of plot. Valid strings are ``{'black_white', 'black_white_dark', 'sketch',
+            'sketch_dark', 'solarized_light', 'solarized_dark', 'default'}``. If no style is specified, the
+            'black_white' style will be used. Setting style does not modify matplotlib global plotting settings.
+            If None, the current matplotlib settings will be used.

```

Comment:
```suggestion
            If ``None``, the current matplotlib settings will be used.
```
Referencing the python ``None``, so it'll look a little clearer using code formatting.